webui/src/App.js

```diff
@@ -479,22 +569,13 @@ function Plot() {
   );
 }
 
-function App() {
-  const [fetched, setFetched] = useState(false);
-  const [profiles, setProfiles] = useState([]);
-  const [activeProfile, setActiveProfile] = useState('');
-
+function FSRWebUI(props) {
+  const { curValuesRef, emit, defaults, wsCallbacksRef } = props;
+  const numPanels = defaults.thresholds.length;
```

Comment:
Can we rename to `numSensors` (that's probably what it should've been named from the start).